### ISSUE TITLE
Delete older files from storage

### ISSUE THREAD

**Issue by Zaied** [webp-sh/webp_server_go#282]
We have been using this service for one of our projects for a while. But the problem storage often gets full and we get errors on the client side. We tried to run a cron job to run the command rm/rf to clean all older files but the issue is clearing too many files shows an error that arguments are too long.

Any way around to solve it?

**Comment by n0vad3v** [comment]
Thanks for feedback, currently you can have a try using `find`, example:
```
0 0 * * * find /path/to/your/webp/exhaust/ -type f -ctime +1 -exec rm {} \;
```

Related issue: https://github.com/webp-sh/webp_server_go/issues/214

**Comment by n0vad3v** [comment]
Hi @Zaied ,We've released version 0.12.0 which contains a configuration called `MAX_CACHE_SIZE` that will automatically clean local files if local cache directories' size are larger than setting value, please give it a shot if you're interested. 😇
https://github.com/webp-sh/webp_server_go/releases/tag/0.12.0

Documentation: https://docs.webp.sh/usage/configuration/
